apps/api/src/app/app.module.ts

```diff
@@ -2,11 +2,17 @@ import { Module } from '@nestjs/common';
 
 import { AppController } from './app.controller';
 import { VMOrchestrationModule } from '../../../../libs/api-implementation/orchestration/src';
-import { ApiImplementationRegressionTestsModule } from '../../../../libs/api-implementation/regression-tests/src'
-import { ApiImplementationDatabaseModule } from '../../../../libs/api-implementation/database/src'
+import { ApiImplementationRegressionTestsModule } from '@new-sample-platform/api-implementation/regression-tests';
```

Comment:
This looks nice, but can you change it so it drops the "new" part? Just "sample-platform" should be sufficient, and "new" will be outdated at some point in the future ;)

Reply:
Yeap, it is doable pretty easily. It depends on the name of the repository. Right now I have an application called `sample-platform` so I guess that will cause some conflict. I think I should change that to the `frontend` or so, and change the name of the repo to `sample-platform`. But I guess that's not the highest priority task, can I postpone it till the more important tasks will be done?

Reply:
If it's not that difficult to achieve, no problem waiting until other more important things are done.

Reply:
Yeah +1 to both of you.

Reply:
A reminder I've created, https://github.com/zelzhan/new-sample-platform/issues/14